packages/eslint-plugin-formatjs/index.ts

```diff
@@ -64,119 +64,131 @@ import {
+  // @ts-expect-error
+  [noUselessMessageName]: noUselessMessage,
+  // @ts-expect-error
+  [preferFormattedMessageName]: preferFormattedMessage,
+  // @ts-expect-error
+  [preferPoundInPluralName]: preferPoundInPlural,
+  // @ts-expect-error
+  [noMissingIcuPluralOnePlaceholdersName]: noMissingIcuPluralOnePlaceholders,
+}
+
+// Base plugin
 const plugin: ESLint.Plugin = {
-  configs: {
-    strict: {
-      plugins: ['formatjs'],
-      rules: {
-        'formatjs/no-offset': 'error',
-        'formatjs/enforce-default-message': ['error', 'literal'],
-        'formatjs/enforce-description': ['error', 'literal'],
-        'formatjs/enforce-placeholders': 'error',
-        'formatjs/no-emoji': 'error',
-        'formatjs/no-multiple-whitespaces': 'error',
-        'formatjs/no-multiple-plurals': 'error',
-        'formatjs/no-complex-selectors': ['error', {limit: 20}],
-        'formatjs/no-useless-message': 'error',
-        'formatjs/prefer-pound-in-plural': 'error',
-        'formatjs/no-missing-icu-plural-one-placeholders': 'error',
-        'formatjs/enforce-id': [
-          'error',
-          {
-            idInterpolationPattern: '[sha512:contenthash:base64:10]',
-          },
-        ],
-        'formatjs/enforce-plural-rules': [
-          'error',
-          {
-            one: true,
-            other: true,
-          },
-        ],
-        'formatjs/no-literal-string-in-jsx': [
-          'error',
-          {
-            props: {
-              include: [['*', '{label,placeholder,title}']],
-            },
+  meta: {name, version},
+  rules,
+}
+
+// Configs
+const configs: ESLint.Plugin['configs'] = {
+  strict: {
+    name: 'formatjs/strict',
+    plugins: {formatjs: plugin},
+    rules: {
+      'formatjs/no-offset': 'error',
+      'formatjs/enforce-default-message': ['error', 'literal'],
+      'formatjs/enforce-description': ['error', 'literal'],
+      'formatjs/enforce-placeholders': 'error',
+      'formatjs/no-emoji': 'error',
+      'formatjs/no-multiple-whitespaces': 'error',
+      'formatjs/no-multiple-plurals': 'error',
+      'formatjs/no-complex-selectors': ['error', {limit: 20}],
+      'formatjs/no-useless-message': 'error',
+      'formatjs/prefer-pound-in-plural': 'error',
+      'formatjs/no-missing-icu-plural-one-placeholders': 'error',
+      'formatjs/enforce-id': [
+        'error',
+        {
+          idInterpolationPattern: '[sha512:contenthash:base64:10]',
+        },
+      ],
+      'formatjs/enforce-plural-rules': [
+        'error',
+        {
+          one: true,
+          other: true,
+        },
+      ],
+      'formatjs/no-literal-string-in-jsx': [
+        'error',
+        {
+          props: {
+            include: [['*', '{label,placeholder,title}']],
           },
-        ],
-        'formatjs/blocklist-elements': ['error', {elements: ['selectordinal']}],
-      },
+        },
+      ],
+      'formatjs/blocklist-elements': ['error', {elements: ['selectordinal']}],
```

Comment:
Oo, the integration tests served their purpose.  Looks like there was an issue with the original config for `blocklist-elements`
```suggestion
      'formatjs/blocklist-elements': ['error', ['selectordinal']],
```
Per
![image](https://github.com/user-attachments/assets/307eb0d9-6c4c-4e8d-b05d-169e1ea63311)

Reply:
Also that screen shot made me realize that the documentation on the site should probably be updated for the new config format